2023-06-29 17:07:28,643 DEBUG   MainThread:3025949 [wandb_config.py:_load_defaults():133] no defaults not found in config-defaults.yaml
2023-06-29 17:07:28,652 DEBUG   MainThread:3025949 [meta.py:_setup_code_git():49] probe for git information
2023-06-29 17:07:28,672 DEBUG   MainThread:3025949 [run_manager.py:__init__():547] Initialized sync for relso-optim/zweeqg6i
2023-06-29 17:07:28,674 INFO    MainThread:3025949 [run_manager.py:wrap_existing_process():1147] wrapping existing process 3025710
2023-06-29 17:07:28,674 WARNING MainThread:3025949 [io_wrap.py:register():104] SIGWINCH handler was not None: <Handlers.SIG_DFL: 0>
2023-06-29 17:07:28,805 INFO    MainThread:3025949 [run_manager.py:init_run():926] system metrics and metadata threads started
2023-06-29 17:07:28,805 INFO    MainThread:3025949 [run_manager.py:wrap_existing_process():1164] informing user process we are ready to proceed
2023-06-29 17:07:28,805 INFO    MainThread:3025949 [run_manager.py:_sync_etc():1296] entering loop for messages from user process
2023-06-29 17:07:29,650 INFO    Thread-3  :3025949 [run_manager.py:_on_file_modified():692] file/dir modified: /home/nrutecka-nlp/ReLSO-with-Linformer/optim_logs/Relso_embed100relso1/TAPE/ns200/2023-06-29-17-07-22/wandb/run-20230629_150722-zweeqg6i/wandb-history.jsonl
2023-06-29 17:07:48,745 INFO    MainThread:3025949 [run_manager.py:_sync_etc():1319] received message from user process: {"exitcode": 0}
2023-06-29 17:07:48,745 INFO    MainThread:3025949 [run_manager.py:_sync_etc():1406] closing log streams and sending exitcode to W&B
2023-06-29 17:07:48,745 INFO    MainThread:3025949 [run_manager.py:shutdown():1070] shutting down system stats and metadata service
cess: {"exitcode": 0}
2023-06-29 17:07:48,745 INFO    MainThread:3025949 [zweeqg6i:run_manager.py:_sync_etc():1406] closing log streams and sending exitcode to W&B
2023-06-29 17:07:48,745 INFO    MainThread:3025949 [zweeqg6i:run_manager.py:shutdown():1070] shutting down system stats and metadata service
